bricksrc/properties.py

```diff
@@ -1,5 +1,5 @@
 from rdflib import Literal
-from .namespaces import A, OWL, RDFS, BRICK, VCARD, UNIT
+from .namespaces import A, OWL, RDFS, BRICK, VCARD, UNIT, TAG
```

Comment:
Does `TAG` get used in `properties.py`? If not, it should be removed

Reply:
Oh, that was leftover from what I was doing the Controller stuff. Will remove it.